src/css/components/header-buttons-mobile.css

```diff
@@ -1,22 +1,34 @@
 .js-overlay-active .mobile-buttons {
   position: fixed;
   top: 1rem;
```

Comment:
You're still declaring the top property twice (1rem here, and 0 below)

Reply:
Removed positions